src/slug.ts

```diff
@@ -0,0 +1,19 @@
+export function slugref(envVar: string): string {
+  return envVar
+    .toLowerCase()
+    .replace(RegExp('refs/(heads|tags)/'), '')
+    .replace(new RegExp('[_/@]', 'g'), '-')
+    .substring(0, 63)
```

Comment:
This implementation miss the trim of the `-` at the end and the begging of the text just before cut it.

Reply:
Do you have an example before and after change that `slugref` should do and not `slug` to add in  tests ?  @rlespinasse

Reply:
fixed